api/v1beta2/ratelimitpolicy_types.go

```diff
@@ -252,6 +252,10 @@ func (r *RateLimitPolicy) BackReferenceAnnotationName() string {
 	return "kuadrant.io/ratelimitpolicies"
 }
 
+func (r *RateLimitPolicy) DirectReferenceAnnotationName() string {
+	return "kuadrant.io/ratelimitpolicy"
```

Comment:
maybe we could move these functions returning static strings to constants in the future